department-of-veterans-affairs/abd-vro
# Store description field into RDS from kafka payload

Associated tickets or Slack threads:
- #2810 

## How does this fix it?[^1]
- Added the @NoLogging annotation.
- Applied the annotation to the description field.
- Overrode the toString method to prevent printing the field value if the @NoLogging annotation is present.
- Made backend changes to store the value of the description field in the database.

## How to test this PR
- verify description field in stored in RDS db


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
